tests/test_anonymize_through_time.py

```diff
@@ -0,0 +1,39 @@
+import pydicom
+import pytest
+
+from itertools import filterfalse, tee
+from typing import Callable, Tuple, Union, ValuesView
+
+from dicognito.anonymizer import Anonymizer
+
+from .data_for_tests import load_minimal_instance
+
+
+@pytest.mark.xfail(raises=AssertionError)
+def test_dataset_anonymizes_same_with_same_seed():
+    with load_minimal_instance() as dataset1:
+        anonymizer = Anonymizer(seed="SOME_FIXED_SEED")
+        anonymizer.anonymize(dataset1)
+    with load_minimal_instance() as dataset2:
+        anonymizer = Anonymizer(seed="SOME_FIXED_SEED")
+        anonymizer.anonymize(dataset2)
+
+    mismatches, matches = partition(lambda value: value == dataset2[value.tag], dataset1.values())
+
+    assert list(value.name for value in matches)
+    assert not list(value.name for value in mismatches)

```

Comment:
Causes the test to fail (which `xfail` makes okay), since the result contains `'Instance Creator UID', 'SOP Instance UID', 'Study Instance UID', 'Series Instance UID', 'Frame of Reference UID'`.